dev-tools/reconfigurator-sp-updater/src/main.rs

```diff
@@ -374,8 +377,40 @@ struct SetArgs {
 #[derive(Clone, Debug, Subcommand)]
 enum Component {
     Sp {
+        /// expected version of the active slot
+        #[arg(long, short = 'a')]
+        expected_active_version: ArtifactVersion,
+        /// expected version of the inactive slot
+        #[arg(long, short = 'i')]
+        expected_inactive_version: ExpectedVersion,
+    },
+    Rot {
+        /// whether we expect the "A" or "B" slot to be active
+        #[arg(long, short = 's')]
+        expected_active_slot: RotSlot,
+        /// expected version of the "A" or "B" slot (as per the
+        /// active slot specified in expected_active_slot)
+        #[arg(long, short = 'a')]
```

Comment:
Take it or leave it: I feel like it'd be easier to use if this were "expected A slot version" and "expected B slot version".  But you've been using this more than I have.

I think it's especially confusing though that the short arg is "-a" -- someone would reasonably think that sets the expected slot A version.

Reply:
I think I ran it so much that it was easier either way for me. But for someone new coming in to the command, I can see why this would be so confusing. Changed!